### PR TITLE
testing/gsoap: new aport

### PR DESCRIPTION
**Pull request by xming**
A cross-platform C and C++ SDK for SOAP/XML Web service
http://gsoap2.sourceforge.net

### CODE REVIEW

File: testing/gsoap/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Contributor: <xmingske@gmail.com>
+# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
```
Comment (Barthalion) [inline testing/gsoap/APKBUILD]: Um, have Natanael agreed on this?

File: testing/gsoap/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	for f in $source; do
+		case $f in
+		*.diff)	patch -p1 < "$srcdir"/$f
+		esac
+	done
+}
+
+build() {
+	cd "${_builddir}"
+	touch configure.ac aclocal.m4 configure Makefile.am Makefile.in
```
Comment (Barthalion) [inline testing/gsoap/APKBUILD]: I'd move that to `prepare()`.

File: testing/gsoap/APKBUILD (hunk 1 above)
Comment (xming) [inline testing/gsoap/APKBUILD]: hmm, I've asked him once if he would mind if I put him as Maintainer in the APKBUILDs and he said no at that time, did the policy change?

File: testing/gsoap/APKBUILD (hunk 2 above)
Comment (xming) [inline testing/gsoap/APKBUILD]: agreed. I will make a new PR when the issue above has cleared.

**Comment by clandmeter** [comment]
This seems already merged, closing.

**Pull request closed without merging**